lib/http-context.js

```diff
@@ -38,6 +41,8 @@ function HttpContext(req, res, method, options) {
   this.args = this.buildArgs(method);
   this.methodString = method.stringName;
   this.supportedTypes = this.options.supportedTypes || DEFAULT_SUPPORTED_TYPES;
+  var streamDesc = method.stream;
+  var methodReturnsStream = !!streamDesc;
```

Comment:
Never mind, I see that `method.stream` is set in shared-method.js. I'll delete these two comments.